tools/copy_partitions.py

```diff
@@ -0,0 +1,102 @@
+"""Copy the partitions to a cluster of machines."""
+import os
+import stat
+import sys
+import subprocess
+import argparse
+import signal
+import logging
+import json
+import copy
+
+def copy_file(file_name, ip, workspace):
+    print('copy {} to {}'.format(file_name, ip + ':' + workspace + '/'))
+    cmd = 'scp -o StrictHostKeyChecking=no ' + file_name + ' ' + ip + ':' + workspace + '/'
```

Comment:
It's much better to use rsync. Rsync can do compression and only sync the difference, which would be more reliable